assets/src/blocks/components/Lightbox/Lightbox.js

```diff
@@ -26,21 +26,19 @@ export const Lightbox = ({index, isOpen, items, onClose = () => {}}) => {
       return;
     }
 
-    const photoSwipe = new PhotoSwipe(photoSwipeElement, PhotoSwipeUI_Default, items, photoSwipeOptions);
-
-    // eslint-disable-next-line no-shadow
-    photoSwipe.listen('gettingData', (index, galleryItem) => {
-      if (galleryItem.w < 1 || galleryItem.h < 1) {
-        const imageSizeHandler = new Image();
-        imageSizeHandler.onload = function() {
-          galleryItem.w = this.width;
-          galleryItem.h = this.height;
-          photoSwipe.updateSize(true);
-        };
-        imageSizeHandler.src = galleryItem.src;
+    const preloadItems = items.map(item => {
```

Comment:
We could use a derived array instead
```suggestion
    const preloadItems = items.map(item => ({
       src: item.originalSrc?.url || item.src
       w: item.originalSrc?.width || item.w,
       h: item.originalSrc?.height || item.h,
       title: item.title,
       }));
```

Reply:
Done, thanks!